CONTRIBUTING.md

```diff
@@ -1,26 +1,10 @@
-# Contributing guidelines
```

Comment:
Why do we still need this file?
Should we just refer the contributing guideline inside the master README.md?

Reply:
Updated the link to the contribution guidelines from CONTRIBUTING.md to wiki for every README.md files. 

I'd like to suggest to keep the CONTRIBUTING.md since it is considered as one of the default community health files for GitHub repo.